<?xml version="1.0"?>
<launch>
    <!-- arguments -->
  <arg name="world_name" default="$(find ros_object_detector)/worlds/test.world"/>
	<arg name="gazebo_gui" default="true"/>

	<!-- Gazebo simulation -->
	<include file="$(find panda_simulation)/launch/simulation.launch" >
		<arg name="world_name" value="$(arg world_name)"/>
		<arg name="gui" value="$(arg gazebo_gui)" />
	</include>

	<!-- Object detection -->
	<node name="ros_object_detector" pkg="ros_object_detector" type="object_detector.py" output="screen">
    <param name="saved_model" value="$(find ros_object_detector)/models/pepper_frcnn"/>
    <param name="label_map" value="$(find ros_object_detector)/label_map_pepper.pbtxt"/>
    <rosparam command="load" file="$(find ros_object_detector)/config/simulation_config.yaml"/>
  </node>

</launch>
